src/test/js/App.test.js

```diff
@@ -0,0 +1,5 @@
+
+
+test('adds 1 + 2 to equal 3', () => {
```

Comment:
Small preference, but I prefer using the more standard approach of "describe" and "it" ("it" is interchangeable with "test" in Jest). The result is that you have something like this:
```
describe('Health tests', () => {
  it('evaluates 1+2', () => {
    ...
  }
})
```
Using describe makes it so that your tests are grouped together under a clear name in the test logs. The reason for using "it" instead of "test" is that it flows like an english sentence, as you can just read it as "It evaluates 1+2" which makes logs and code more readable.